plugin/trino-accumulo/src/test/java/io/trino/plugin/accumulo/TestAccumuloConnectorTest.java

```diff
@@ -73,21 +79,21 @@ protected TestTable createTableWithDefaultColumns()
     }
 
     @Override
-    public void testCreateSchema()
+    public void testAddColumn()
     {
-        // schema creation is not supported
+        throw new SkipException("Adding columns via SQL are not supported until adding columns with comments are supported");
     }
 
     @Override
-    public void testAddColumn()
+    public void testDropColumn()
     {
-        // Adding columns via SQL are not supported until adding columns with comments are supported
+        throw new SkipException("Dropping columns are not supported by the connector");
```

Comment:
nit: avoid reordering methods as that makes the diffs and history confusing.

If you add `testRenameTableAcrossSchema` after `testCreateTableAsSelect` Git would have an easier time making sense of what actually changes.

Reply:
Sure. I'll pay attention to this aspect also in the future PRs.